requirements/readthedocs.txt

```diff
@@ -11,4 +11,4 @@ pydot
 # linting tools
 pylint-django
 twine
-pyOpenSSL
+pyOpenSSL
```

Comment:
Not sure what changed here but it definitely unrelated. Revert the change.

Reply:
Actually I tried to add textdistance module here but removed later in next push maybe that's why it is showing to be changed